lib/checkout_helper.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+class CheckoutHelper
```

Comment:
this is really good but needs a bit of reorganizing:
- The only methods on `self` should be `check_in!` and ` check_out!`, which should read like this:
```ruby
def self.check_out!(**params)
  new(**params).check_out!
end
```
- There should be an `initialize` method that sets some instance variables so you don't have to pass them around. It should use keyword args.
- Any method that can raise an un-rescued exception should end in a `!`